subsys/net/l2/openthread/Kconfig.thread

```diff
@@ -144,3 +144,11 @@ config OPENTHREAD_IP6_MAX_EXT_MCAST_ADDRS
 	int "The maximum number of supported IPv6 multicast addresses allows to be externally added"
 	range 0 32
 	default 2
+
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/37904 is merged, please update to be _[nrf fromtree] net: ..._

Reply:
Done, thanks for checking.